torch/tensor.py

```diff
@@ -362,6 +362,12 @@ def __rfloordiv__(self, other):
     __ge__ = _C._TensorBase.ge
     __abs__ = _C._TensorBase.abs
 
+    def __radd__(self, other):
+        if isinstance(self, torch.Tensor):
+            if (self.dtype == torch.bool or self.dtype == torch.uint8):
+                return torch.sum(self) + other
```

Comment:
What happens if you do `5 + x`, where `x` is a bool tensor?

Reply:
More specifically, if x has more than 1 element, I think this doesn't work as intended

Reply:
yep, it's also concerning that this incorrect implementation caused no test errors...